[2021-12-25 11:18:58,393] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: survey.myfunction 2021-12-25T11:18:55.821072+00:00 [queued]>
[2021-12-25 11:18:58,421] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: survey.myfunction 2021-12-25T11:18:55.821072+00:00 [queued]>
[2021-12-25 11:18:58,422] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-12-25 11:18:58,423] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-12-25 11:18:58,423] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-12-25 11:18:58,446] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): myfunction> on 2021-12-25T11:18:55.821072+00:00
[2021-12-25 11:18:58,452] {standard_task_runner.py:52} INFO - Started process 42 to run task
[2021-12-25 11:18:58,460] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'survey', 'myfunction', '2021-12-25T11:18:55.821072+00:00', '--job-id', '182', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/survey.py', '--cfg-path', '/tmp/tmpoc5xa3cp', '--error-file', '/tmp/tmpkd7l9s19']
[2021-12-25 11:18:58,460] {standard_task_runner.py:77} INFO - Job 182: Subtask myfunction
[2021-12-25 11:18:58,530] {logging_mixin.py:104} INFO - Running <TaskInstance: survey.myfunction 2021-12-25T11:18:55.821072+00:00 [running]> on host e1668ec19f6d
[2021-12-25 11:18:58,594] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=survey
AIRFLOW_CTX_TASK_ID=myfunction
AIRFLOW_CTX_EXECUTION_DATE=2021-12-25T11:18:55.821072+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-25T11:18:55.821072+00:00
[2021-12-25 11:18:59,226] {python.py:118} INFO - Done. Returned value was: None
[2021-12-25 11:18:59,257] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=survey, task_id=myfunction, execution_date=20211225T111855, start_date=20211225T111858, end_date=20211225T111859
[2021-12-25 11:18:59,419] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-12-25 11:18:59,489] {local_task_job.py:146} INFO - Task exited with return code 0
